src/Writing/Writer.php

```diff
@@ -95,12 +95,19 @@ protected function writeOpenAPISpec(array $parsedRoutes): void
             c::info('Generating OpenAPI specification');
 
             $spec = $this->generateOpenAPISpec($parsedRoutes);
+
+            if (config('scribe.openapi.json', false)) {
+                $fileContent = json_encode($spec, JSON_PRETTY_PRINT);
+            } else {
+                $fileContent = Yaml::dump($spec, 20, 2, Yaml::DUMP_EMPTY_ARRAY_AS_SEQUENCE | Yaml::DUMP_OBJECT_AS_MAP);
+            }
```

Comment:
In my personal opinion, writing this part of the logic in generateOpenAPISpec as the return result of a string would be more intuitive, and I am not sure if it is feasible.